addons/website_livechat/__manifest__.py

```diff
@@ -32,6 +32,7 @@
             'website_livechat/static/src/js/**/*',
         ],
         'web.assets_frontend': [
+            'website/static/src/**/common/**/*',
```

Comment:
This should not be in the `assets_frontend` here either, which should stay minimal.

What we want is to target the live chat bundle that defines the store, which I believe is `im_livechat.assets_embed_core`.

Reply:
Done.